packages/terra-clinical-header/src/Header.jsx

```diff
@@ -22,17 +22,24 @@ const propTypes = {
    * Content to be displayed at the end of the header
    */
   endContent: PropTypes.element,
+
+  /**
+   * A Boolean indicating if element is a subheader. Default is false
```

Comment:
Nitpick.

Do we need to specify default is false in the prop documentation? I believe our table will latch onto the defaultProps and get the value from there.